g11n-ws/vip-manager-l10n-agent/src/main/java/com/vmware/l10agent/utils/HttpRequester.java

```diff
@@ -385,6 +386,7 @@ public boolean verify(String hostname, SSLSession session) {
 				httpsConn.setSSLSocketFactory(getSocketFactory());
 			}
 		} catch (IOException e) {
+		   throw e;
```

Comment:
Don't try this exception.

Reply:
this is private method, why don't show this exception